installer/helm/chart/volcano/values.yaml

```diff
@@ -210,6 +210,9 @@ custom:
   # Specify agent cni config path.
   agent_cni_config_path: /etc/cni/net.d/cni.conflist
 
+  # Specify feature gates for components
```

Comment:
Do we need this?

Reply:
DRA plugin has two feature gates `DynamicResourceAllocation` and `DRAAdminAccess`, if we need to enable the DRA plugin, the feature gate `DynamicResourceAllocation` must be enabled, therefore we need to add command flag for scheduler